picocontainer/pom.xml

```diff
@@ -35,16 +35,50 @@
 			<plugin>
 				<groupId>org.apache.felix</groupId>
 				<artifactId>maven-bundle-plugin</artifactId>
-				<version>2.3.4</version>
+				<version>3.5.0</version>
 				<extensions>true</extensions>
 				<configuration>
 					<instructions>
 						<Bundle-SymbolicName>${project.groupId}.${project.artifactId}</Bundle-SymbolicName>
 						<Bundle-Name>${project.name}</Bundle-Name>
+						<Embed-Dependency>javax.inject</Embed-Dependency>
 						<Import-Package>*</Import-Package>
 						<Export-Package>
-							org.picocontainer;version=1.2
+							org.picocontainer;version=1.3,
```

Comment:
❓ would it be easier to make `1.3` a version property so each line wouldn't have to be updated going forward?

Reply:
I just used replace-all, but I could make it a property.

Reply:
meh, this is fine for this PR 👍